### ISSUE TITLE
WASAPI: Issue with Asus Xonar and Stereo mode (multiple channels)

### ISSUE THREAD

**Issue by jaw** [thestk/rtaudio#254]
Asus xonar card is set to "Stereo". (2 channels).
All reverb/compression/eq is turned off.

When the code calls this:
```cpp
    hr = captureAudioClient->GetMixFormat( &captureFormat );
```
The captureFormat.numChannels is equal to 8 (with corresponding multiples in the other fields).

When calling this:
```cpp
      hr = captureAudioClient->Initialize( AUDCLNT_SHAREMODE_SHARED,
                                           loopbackEnabled ? AUDCLNT_STREAMFLAGS_LOOPBACK : AUDCLNT_STREAMFLAGS_EVENTCALLBACK,
                                           0,
                                           0,
                                           captureFormat,
                                           NULL );
```
(Note: loopbackEnabled evaluates to AUDCLNT_STREAMFLAGS_LOOPBACK  in this case as it's an output device)

hr is equal to 0x88890008 a.k.a. "Unknown HRESULT".

Same bug has been reported with realtek on board soundcard.

**Comment by MarcusTomlinson** [comment]
@jaw, I recall you saying this may be a driver issue rather than something in RtAudio a while back. Do you have any more info on this?

**Comment by jaw** [comment]
It is rt audio because other programs work. And it is a MAJOR issue.

**Comment by jaw** [comment]
I think this bug can't be fixed on RtAudio Level, see my longer comment in #281.
TLDR: For now, run WASAPI in its own process to work around the Asus buggy drivers.
